apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx

```diff
@@ -429,6 +430,24 @@ export const AuthenticatorCreateWizardFactory: FunctionComponent<AuthenticatorCr
                     />
                 )
                 : null;
+        case IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.HYPR:
+            return (showWizard && !isEmpty(selectedTemplateWithUniqueName))
+                ? (
```

Comment:
Check if we can use `&& `instead.

Reply:
addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

Reply:
Seems like using && for conditional rendering is an anti-pattern.  

Ref: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
+1. `&&` will result in an `undefined` to be returned as the root element of the component. Which will break React.

https://github.com/wso2/identity-apps/blob/master/docs/write-code/DEVELOPING_REACT_CODE.md#conditional-rendering

Reply:
ack. will check other places as well